src/exception/call_stack.cr

```diff
@@ -163,52 +169,57 @@ struct Exception::CallStack
     end
   end
 
-  private def decode_backtrace
+  def printable_backtrace : Array(String)
     show_full_info = ENV["CRYSTAL_CALLSTACK_FULL_INFO"]? == "1"
 
+    frames.compact_map do |frame|
+      function = frame.name || "??"
+      function = frame.sname if show_full_info && frame.sname
+
+      if file = frame.file
+        if (line = frame.line) && (column = frame.column)
+          position = ":#{line}:#{column}"
+        elsif line
+          position = ":#{line}"
+        end
+        line = "#{file}#{position} in '#{function}'"
+      else
+        line = function
+      end
+
+      if show_full_info
+        line = "#{line} at 0x#{frame.ip_address.to_s(16)}"
+      end
```

Comment:
I'm wondering if this stringification code would better live in an instance method on `Frame`?

Reply:
Good catch, will move it there.

Reply:
Yeah, that env var should stay here and be passed as an argument.